client/src/components/FieldHolder/FieldHolder.js

```diff
@@ -42,6 +42,7 @@ function fieldHolder(Field) {
           field: true,
           [this.props.extraClass]: true,
           readonly: this.props.readOnly,
+          'form-group': true,
```

Comment:
https://getbootstrap.com/docs/5.0/migration/#sass

> Dropped form-specific layout classes for our grid system. Use our grid and utilities instead of `.form-group`, `.form-row`, or `.form-inline`.

It would require a non-trivial amount of effort to do that refactor and I like how semantic these class names are, so I opted instead to port those styles into `client/src/styles/_bootstrap-form.scss` instead.

Reactstrap correctly stopped adding these css classes, so I've added them back where necessary.